### PR TITLE
(fix) Stock/Waiver List, bank balance functionality

### PR DESCRIPTION
**Pull request by scottanderson1130**

### CODE REVIEW

File: src/components/Waivers/WaiversList.jsx [hunk 1]
```diff
@@ -28,6 +28,12 @@ function WaiversList({
 
   const league = useSelector(selectLeague);
 
+  useEffect(() => {
+    axios.get(`/stock/bank/${user.id}/${league}`)
+      .then((response) => setBankBalance(response.data.bank_balance));
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (jstrzesz) [inline src/components/Waivers/WaiversList.jsx]: why are we using eslint-disable here?

File: src/components/Waivers/WaiversList.jsx [hunk 2]
```diff
@@ -28,6 +28,12 @@ function WaiversList({
 
   const league = useSelector(selectLeague);
 
+  useEffect(() => {
+    axios.get(`/stock/bank/${user.id}/${league}`)
```
Comment (jstrzesz) [inline src/components/Waivers/WaiversList.jsx]: remove this from useEffect() and create a new method outside of the hook. Then call that function inside useEffect

File: src/components/Waivers/WaiversList.jsx [hunk 3]
```diff
@@ -89,8 +95,7 @@ function WaiversList({
           <strong>Bank Balance: </strong>
```
Comment (jstrzesz) [inline src/components/Waivers/WaiversList.jsx:89]: don't use strong tags, instead use css for emphasis

File: src/components/YourStocks/StocksList.jsx [hunk 4]
```diff
@@ -70,7 +70,6 @@ function StocksList({
   const handleSharesSubmit = ((e) => {
     setSharesInput(e.target.value);
   });
-  console.log('BANK BALANCE', bankBalance);
   return (
     <>
```
Comment (jstrzesz) [inline src/components/YourStocks/StocksList.jsx:74]: don't use empty tags. Either add div to the tag or just remove them entirely

File: src/views/YourStocks.jsx [hunk 5]
```diff
@@ -29,7 +29,7 @@ function YourStocks() {
   }, [dispatch, user]);
 
   useEffect(() => {
-    axios.get(`/stock/bank/${user?.id}`)
+    axios.get(`/stock/bank/${user.id}/${league}`)
```
Comment (jstrzesz) [inline src/views/YourStocks.jsx:32]: move this function to a separate method and then call that method inside useEffect

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

File: src/components/YourStocks/StocksList.jsx (hunk 4 above)
Comment (scottanderson1130) [reply src/components/YourStocks/StocksList.jsx:74]: Tables cannot appear as a child of <div> and you must have a parent element for .jsx. So the only option to not get an error is to have <React.Fragment>.

**Review by ChristopherStumpe — APPROVED** [review]

File: src/views/Waivers.jsx [hunk 6]
```diff
@@ -25,17 +24,16 @@ function Waivers() {
       return waiversResponse;
     }
     fetchWaivers();
-  }, [dispatch, user.leagueInfo]);
+  }, [dispatch, league, user.leagueInfo]);
 
   useEffect(() => {
-    axios.get(`/stock/bank/${user.id}`)
+    axios.get(`/stock/bank/${user.id}/${league}`)
```
Comment (jstrzesz) [inline src/views/Waivers.jsx:30]: move this to a separate method and then call it inside useEffect

**Review by josesrodriguez610 — APPROVED** [review]

**Comment by scottanderson1130** [comment]
@jstrzesz Are we good to merge this PR?

**Pull request merged**